experimental/dds/tree2/src/test/rebaserAxiomaticTests.ts

```diff
@@ -225,37 +232,50 @@ export function runExhaustiveComposeRebaseSuite<TContent, TChangeset>(
 
 							for (let i = 0; i < sourceEdits.length; i++) {
 								const edit = sourceEdits[i];
-								const editsToRebaseOver = [
+								const rebasePath = [
 									...inverses.slice(sourceEdits.length - i),
 									editToRebaseOver,
 									...rebasedEditsWithoutCompose,
 								];
 								rebasedEditsWithoutCompose.push(
-									rebaseTagged(edit, ...editsToRebaseOver),
+									rebaseTagged(edit, rebasePath, [editToRebaseOver]),
```

Comment:
I'm confused about what `rebaseTagged` is supposed to do.

Reply:
The same thing that [rebaseOverChanges](https://github.com/microsoft/FluidFramework/blob/main/experimental/dds/tree2/src/test/feature-libraries/sequence-field/utils.ts#L141-L160) is, but more generic?